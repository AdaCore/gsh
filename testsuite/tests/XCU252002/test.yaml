title: Handling of * special parameter
