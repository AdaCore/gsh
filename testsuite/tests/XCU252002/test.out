== no args ==
Quoted *
param: ''
NonQuoted *
== empty args ==
Quoted *
param: ''
NonQuoted *
== three args ==
Quoted *
param: 'a b c'
NonQuoted *
param: 'a'
param: 'b'
param: 'c'
== args with space ==
Quoted *
param: 'a b c d'
NonQuoted *
param: 'a'
param: 'b'
param: 'c'
param: 'd'
